src/Tizen/Tizen.Common/IWindowProvider.cs

```diff
@@ -1,9 +1,25 @@
-﻿using System;
+﻿/*
```

Comment:
I'm worried that moving the interface can break an application. Am I correct that application would usually not use that interface directly but rather just use what is provided by that interface?
Even if your analysis suggests that its safe to remove the interface from old location please leave an empty interface there that would hint everybody triyng to use it what is going on:

```
[Obsolete("Please use Tizen.Common.IWindowProvider instead.", true)]
public interface IWindowProvider {}
```

Reply:
@wiertel 
Originally, I was planning to leave the Multimedia part of the code initially and attach the "obsolete" attribute as you suggested. However, the ACR added to Mutimedia is API12, and now the patch is also API12, so it will be even more confusing. I talked to Mr. Gwon about this.

@hsgwon 
Hello, could you check Mr. @wiertel 's comment? he wants to leave the previously added Multimedia's `IWindowProvider` and add a guide for users explicitly.

Reply:
@wiertel 
Multimedia IWindowInterface has been ACR added in Tizen_9.0 (API12), and the Common IWindowInterface that is about to be modified will also be included in Tizen_9.0 (API12). Therefore, there are no cases where it is used in the APP. Of course, we can leave a blank interface in Multimedia as you suggested. However, if we do this, it may cause an ambiguous error because the same-named interface will be created in the same version, which may confuse the user. Do you think it is a good idea to do so?

Reply:
That's a perfectly valid argument. If the changed interface is a new addition in development branch then there is no issue.